sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -326,18 +333,6 @@ export interface ListPropertiesOfKeysOptions extends coreHttp.OperationOptions {
 export interface ListPropertiesOfKeyVersionsOptions extends coreHttp.OperationOptions {
 }
 
-// @public
```

Comment:
I believe we decided to get rid of LocalCrpytoClient and add a constructor to `CryptographyClient` that takes a `JsonWebKey` and consolidate them.

Reply:
That's the prototype idea, yes.